tests/test_factory.py

```diff
@@ -474,11 +474,21 @@ def test_pyfloat_in_range(self):
         # tests for https://github.com/joke2k/faker/issues/994
         factory = Faker()
 
-        factory.seed_instance(5)
-        result = factory.pyfloat(min_value=0, max_value=1)
-
-        assert result >= 0.0
-        assert result <= 1.0
+        for i in range(20):
+            for min_value,max_value in [
```

Comment:
add a space: `fro min_value, max_value in [`

Reply:
thanks !